tests/strategy/test_wrapping_strategy.py

```diff
@@ -0,0 +1,27 @@
+from unittest import mock
+from yapapi.strategy import WrappingMarketStrategy
+
+
+class MyBaseStrategy(mock.Mock):
+    some_other_value = 123
+
+
+class MyStrategy(WrappingMarketStrategy):
+    some_value = 456
+
+    def some_method(self):
+        return True
+
+
+def test_wrapping_strategy():
+    base_strategy = MyBaseStrategy()
+    strategy = MyStrategy(base_strategy)
+
+    assert strategy.some_value == MyStrategy.some_value
+    assert strategy.some_other_value == MyBaseStrategy.some_other_value
```

Comment:
Please add a case where `MyStrategy` has a method/attribute with the same name as in `MyBaseStrategy` (--> and thus overrides it).

Reply:
good point

Reply:
actually, they don't inherit from each other so, the existent tests already test that "override" ... but I added `some_value` on the base class just to make the point